docs/pages/en-us/Architecture.md

```diff
@@ -5,78 +5,224 @@ title: Architecture
-
-A Mesos/Marathon environment runs the Audit Management web service and Kafka-Vertica Scheduler components, and provides redundancy for these services.
+Auditing is built on Elasticsearch for the messaging and storage of the audit events. Elasticsearch offers high availability, throughput, scalability, and performance to the overall solution. Additionally, Elasticsearch offers strong data analytics capabilities.
 
 ### Audit Management Component Architecture
 
-The figure below illustrates the overall flow and relationship of components in the Audit Management service.
+The figure below illustrates the overall flow and relationship of components in the Audit service.
 
-![Architecture](images/AuditManagementArchitectureDraft.png)
+![Architecture](images/AuditElasticArchitecture.png)
 
-1. Setting up your application for Auditing requires defining an audit event definition XML file. The file is used for:
-	- Generation of the client-side audit library.
-	- Registration for auditing on the server-side.
-2. Using the caf-audit-maven-plugin, the client-side Java library is generated from the audit event definition XML file.
-3. The audit event definition XML containing the application and its events for auditing is registered with the server-side Audit Management web service API's POST /applications endpoint.
-	1. The Audit Management web service creates application and event schema based on the audit event definition XML.
-4. A tenant with a list of audited applications is registered with the server-side Audit Management web service API's POST /tenants endpoint.
-	1. The web service creates tables for each application with which the tenant is registered in Vertica for storage of the tenant's application events.
-	2. The web service associates a topic for each application that the tenant is registered with in the Kafka-Vertica Scheduler.
-5. The audited application makes calls to the generated client-side library to send tenant audit events to Kafka messaging.
-	1. Kafka receives messages from the client-side library and stores them in per application, per tenant topics.
-	2. Kafka-Vertica Scheduler listens to the topics that are registered with it and streams them into the tenant's application table in Vertica.
+1. Setting up your application for Auditing requires defining an audit event definition XML file. The file is used for generation of the client-side audit library.
+2. Using the `caf-audit-maven-plugin`, the client-side Java library is generated from the audit event definition XML file.
+3. The audited application makes calls to the generated client-side library to send tenant audit events to Elasticsearch which stores them in per tenant indices.
```

Comment:
Point 3. is confusing.

"An application being audited uses the generated client-side library to create and send audit events to Elasticsearch. 
 A new index is created per tenant in the Elasticsearch.  Therefore the audit events are stored per tenant within Elasticsearch."

Something like the above.  Although I am not sure that is the style you are looking for or if it covers enough detail.